src/test/java/bookbob/BookBobTest.java

```diff
@@ -137,8 +145,8 @@ void test_listOnePatient_inRecord() throws IOException {
                         "v/21-10-2024 15:48 al/Pollen s/Female mh/Diabetes", records);
         command.list(records);
         assertEquals("Patient James-Ho with NRIC S9534567A added.\n" +
-                        "Name: James-Ho, NRIC: S9534567A, Phone: 91234567, Address: NUS-PGPR, DOB: 13121995, " +
-                        "Allergy: Pollen, Sex: Female, Medical History: Diabetes",
+                        "Name: James-Ho, NRIC: S9534567A, Phone: 91234567, Home Address: NUS-PGPR, DOB: 13121995, " +
+                        "Allergies: [Pollen], Sex: Female, Medical Histories: [Diabetes], Visit Date: 21-10-2024 15:48",
```

Comment:
Good that you changed allergies and medical histories into an ArrayList instead of the original String